### PR TITLE
feat: add auto number for mathjax

### PR DESCRIPTION
**Pull request by panda-z**

### CODE REVIEW

**Comment by olOwOlo** [comment]
Hi, thank you for your pull request.
Can I use Chinese?

- 我对 mathjax 不怎么了解，查询了下[文档](http://docs.mathjax.org/en/latest/tex.html#automatic-equation-numbering)，看起来  `extensions` 不是必要的？它们是必要的吗？
- 因为 #32 的原因，有如下的问题

    snipaste_20180404_225520

    ```
    \begin{equation}
      \int_0^\infty \frac{x^3}{e^x-1}\,dx = \frac{\pi^4}{15}
      \label{eq:sample}
    \end{equation}
    ```

    是我的姿势不对？

**Comment by panda-z** [comment]
Removed extensions.

**Comment by panda-z** [comment]
The extra horizontal scroll bar is due to an incorrect value of `margin-left` of `mjx-itable`. A workaround is loop over all `mjx-itable` and modify their `margin-left`.

**Comment by panda-z** [comment]
@olOwOlo I don't see the extra horizontal scroll bar in Firefox. However, it appears in Chromium.

**Comment by olOwOlo** [comment]
> I don't see the extra horizontal scroll bar in Firefox. However, it appears in Chromium.

Me too...

For this theme, there is a workaround
```css
.post .post-content .mjx-full-width {
  width: 759px;
}
```

**Comment by panda-z** [comment]
Another workaround is adding `padding-right: 1px;` for `.MJXc-display`
```
   .MJXc-display {
     overflow-x: auto;
     overflow-y: hidden;
+    padding-right: 1px;
   }
```
You may want to modify `_content.scss` to avoid the redundant horizontal scroll bar in Chrome/Chromium.
